### ISSUE TITLE
gitbook-plugin-github specified twice in dependencies

### ISSUE THREAD

**Issue by xPaw** [sysgears/mochapack#40]
Once in dev, and once in normal.

After upgrading mochapack, we get this warning:
```
warning gitbook-plugin-github@2.0.0: The engine "gitbook" appears to be invalid.
```

which leads me to believe it doesn't need to be in `dependencies`.

This commit added it to dependencies: https://github.com/sysgears/mochapack/commit/7d0cbaff91b4f0b91b3e27afb342a92bea567df3 and this one downgraded it: https://github.com/sysgears/mochapack/commit/b9a86aaaf07abcf9c5b75ccdc015042c4a33452f

**Comment by larixer** [comment]
@xPaw Hm, yes, looks like it was added to dependencies by mistake in this PR: #37

**Comment by larixer** [comment]
@xPaw Published `mochapack@1.1.9` with the fix

**Comment by xPaw** [comment]
Thanks!

**Comment by xPaw** [comment]
The version published to npm does not match what's on github, and thus gitbook dependency is still there: https://app.renovatebot.com/package-diff?name=mochapack&from=1.1.8&to=1.1.9

**Comment by larixer** [comment]
@xPaw Hmm, looks like this dep is auto-added on publish. Trying to figure out why...

**Comment by larixer** [comment]
@xPaw Published via yarn `@1.1.11`, looks like it doesn't have this dep. I'm not sure why npm were adding it on publish
